web_src/js/components/DashboardRepoList.vue

```diff
@@ -76,7 +76,7 @@ const sfc = {
       return this.repos.length > 0 && this.repos.length < this.counts[`${this.reposFilter}:${this.archivedFilter}:${this.privateFilter}`];
     },
     searchURL() {
-      return `${this.subUrl}/repo/search?sort=updated&order=desc&uid=${this.uid}&team_id=${this.teamId}&q=${this.searchQuery
+      return `${this.subUrl}/repo/search?sort=alpha&order=asc&uid=${this.uid}&team_id=${this.teamId}&q=${this.searchQuery
```

Comment:
Can this be configurable? Sorting by updated time is helpful to me. I usually work on a fixed number of repos at a time, and having them at the top is helpful. The first few repos sorted alphabetically may not be active.

Reply:
Yeah, I'm also not sure about alphabetic sorting here:
I've gotten used to seeing my current repos there as well on the first page to navigate to them quickly.
I don't think it's a good idea to require alphabetic sorting here by default.